### PR TITLE
Made energy-kJ and energy-kcal have only one unit, removed select

### PR DESCRIPTION
**Pull request by VaiTon**
### Before
immagine
immagine
immagine

### After
immagine

### CODE REVIEW

File: cgi/product_multilingual.pl [hunk 1]
```diff
@@ -1876,82 +1876,79 @@ ($$$$$$)
...
 ;
 
-		if ($nid ne 'alcohol') {
+		if (($nid eq 'alcohol') or ($nid eq 'energy-kj') or ($nid eq 'energy-kcal')) {
+			my $unit = '';
+			
+			if (($nid eq 'alcohol')) { $unit = '% vol / °' } # alcohol in % vol / °
+			elsif (($nid eq 'energy-kj')) { $unit = 'kJ' }
+			elsif (($nid eq 'energy-kcal')) { $unit = 'kcal'}
+
```
Comment (VaiTon) [inline cgi/product_multilingual.pl]:
```suggestion
```

File: cgi/product_multilingual.pl [hunk 2]
```diff
@@ -1876,82 +1876,78 @@ ($$$$$$)
 HTML
 ;
 
-		if ($nid ne 'alcohol') {
+		if (($nid eq 'alcohol') or ($nid eq 'energy-kj') or ($nid eq 'energy-kcal')) {
+			my $unit = '';
+			
+			if (($nid eq 'alcohol')) { $unit = '% vol / °' } # alcohol in % vol / °
+			elsif (($nid eq 'energy-kj')) { $unit = 'kJ' }
+			elsif (($nid eq 'energy-kcal')) { $unit = 'kcal'}
```
Comment (hangy) [inline cgi/product_multilingual.pl]: Does this work without semicolons terminating each line?

Comment (VaiTon) [reply cgi/product_multilingual.pl]: >Every simple statement must be terminated with a semicolon, unless it is the final statement in a block, in which case the semicolon is optional. 

But for the sake of convention i will add them. :)

Comment (hangy) [reply cgi/product_multilingual.pl]: Oh okay, that's news to me. 🙂 nvm then.

**Pull request merged**